src/decision/dMCDA.jl

```diff
@@ -737,87 +721,111 @@ function guided_site_selection(
 end
 
 """
-    distance_sorting(pref_locs::AbstractArray{Int}, s_order::Matrix{Union{Float64,Int64}}, dist::Matrix{Float64}, min_dist::Float64, rankings::Matrix{Int64}, rank_col::Int64)::Tuple{Vector{Union{Float64,Int64}},Matrix{Int64}}
-
-Find selected locations with distances between each other < median distance-dist_thresh*(median distance).
-Replaces these locations with those in the top ranks if the distance between these is greater.
+    constrain_reef_cluster(reefs::Union{Vector{String},Vector{Float64}}, s_order::Matrix{Union{Float64,Int64}}, rankings::Matrix{Int64},
+    area_to_seed::Float64, available_space::Vector{Float64}, n_site_int::Int64, n_reefs::Int64)
 
 # Arguments
-- `pref_locs` : Original n highest ranked locations selected for seeding or shading.
-- `s_order` : Current order of ranked sites in terms of numerical site ID.
-- `dist` : Matrix of unique distances between sites.
-- `min_dist` : Minimum distance between sites for selected sites.
-- `rankings` : Ranking data
-- `rank_col` : Index of column holding location ranks
+- `reefs` : List of the the reefs each location sits within
+- `s_order` : Ordered set of locations and their aggregate criteria score
+- `rankings` : Current ranks of the set of locations
+- `area_to_seed` : absolute area to be seeded (m²)
+- `available_space` : absolute area available at each location (m²)
+- `n_site_int` : Minimum number of sites to intervene at
+- `n_reefs` : Number of selected locations to allow in the same reef.
 
 # Returns
-New set of selected sites for seeding or shading.
+Tuple :
+    - `pref_locs` : Vector, Indices of preferred intervention locations
+    - `rankings` : Matrix[n_sites ⋅ 3] where columns are site_id, seeding_rank, shading_rank
+        Values of 0 indicate sites that were not considered
```

Comment:
No indentation is necessary, and leads to incorrect formatting when the docs are generated

```suggestion
Tuple :
- `pref_locs` : Vector, Indices of preferred intervention locations
- `rankings` : Matrix[n_sites ⋅ 3] where columns are site_id, seeding_rank, shading_rank
    Values of 0 indicate sites that were not considered
```